rviz_mesh_tools_plugins/include/rviz_mesh_tools_plugins/MeshDisplay.hpp

```diff
@@ -420,8 +421,10 @@ private Q_SLOTS:
   /// Property to set faces transparency
   rviz_common::properties::FloatProperty* m_facesAlpha;
 
-  /// Property to handle topic for vertex colors
+  /// Properties to handle topic for vertex colors
   rviz_common::properties::RosTopicProperty* m_vertexColorsTopic;
+  rviz_common::properties::QosProfileProperty* m_vertexColorsTopicQos;
```

Comment:
(optional)
Hm, didn't occur to me in the first review, but the naming of all `QosProfileProperty` members is slightly strange.
Per thing, there are
* `rviz_common::properties::RosTopicProperty* m_${THING}Topic` -> ui element for picking topic name
* `rviz_common::properties::QosProfileProperty* m_${THING}TopicQoS` -> ui element for picking qos profil
* `rclcpp::QoS* m_${THING}Qos` -> the current qos profile

I feel like `Topic` should not be there in `${THING}TopicQoS`. Maybe call the `QoSProfileProperty`s 
```suggestion
  rviz_common::properties::QosProfileProperty* m_vertexColorsQosPicker;
```
or something similar.

Reply:
I was thinking the same. There are also other names that can do better :D. For example, m_vertexColorsTopic might be better named into m_vertexColorsTopicPicker. I propose to address this issue in another PR.